metron-analytics/metron-profiler/src/main/java/org/apache/metron/profiler/bolt/ProfileBuilderBolt.java

```diff
@@ -246,6 +247,8 @@ private void flush(Tuple tickTuple) {
     // clear the execution state to prepare for the next window
     executor.clearState();
 
+    profileConfig.getTickUpdate().forEach((var, expr) -> executor.assign(var, expr, new HashMap<>()));
```

Comment:
@nickwallen My intent here is to add a hook in the profiler on tick.  Would you take a look and make sure I'm doing this right?  I'm interested specifically in the empty hashmap here.  I'd like to ensure that the current message is being passed in on tick, but I fear it won't be the case.